tools/sdk-automation-core/.gitignore

```diff
@@ -0,0 +1,5 @@
+lib/
+node_modules/
```

Comment:
We should eliminate tool specific gitignore and add things to the root one as needed.

Reply:
Consolidated to the gitignore under the root folder.